intra_process_demo/src/image_pipeline/camera_node.cpp

```diff
@@ -16,6 +16,8 @@
 
 #include <rclcpp/rclcpp.hpp>
```

Comment:
I'd vote to merge as is and ticket the annoyance. For the long term solution I'd like to see cpplint.py patched to allow something like this:

```c++
#include <memory>

#include <rclcpp/rclcpp.hpp>

#include "foo/foo.hpp"
```